multichain-testing/tools/sleep.ts

```diff
@@ -0,0 +1,7 @@
+import type { LogFn } from 'ava';
+
+export const sleep = (ms: number, log?: LogFn) =>
+  new Promise(resolve => {
+    if (log) log(`Sleeping for ${ms}ms...`);
+    setTimeout(resolve, ms);
+  });
```

Comment:
consider simpler body by giving a default log
```suggestion
export const sleep = (ms: number, log: LogFn = () => {}) =>
  new Promise(resolve => {
    log(`Sleeping for ${ms}ms...`);
    setTimeout(resolve, ms);
  });
```